src/java/fr/paris/lutece/plugins/suggest/utils/SuggestUtils.java

```diff
@@ -222,7 +222,7 @@ public static void sendNotificationNewSuggestSubmit( Suggest suggest, SuggestSub
         }
         catch( Exception e )
         {
-            AppLogService.error( "Error during Notify new suggest submit  : " + e.getMessage( ) );
+            AppLogService.error( "Error during Notify new suggest submit  : ", e.getMessage( ) );
```

Comment:
This seems incorrect because you are calling the parametric form of AppLogService.error, but there are no parameter placeholder in the message. You should really pass the exception itself, so that a stack trace is logged and the exception can be diagnosed in the logs

Reply:
Ok, I'll directly pass the Exception as argument.